patches/chrome-installer-mini_installer-mini_installer.cc.patch

```diff
@@ -116,22 +116,26 @@ index 1f39abbeb2579098a54fa82612199f0307ea03b4..6aa317767ed09e31296c5c9af187481a
 +}
 +
 +bool GetStubInstallerFilename(PathString& installer_filename) {
-+  wchar_t value[MAX_PATH];
-+  if (RegKey::ReadSZValue(HKEY_CURRENT_USER, L"Software\\BraveSoftware\\Promo",
-+                          L"StubInstallerPath", value, _countof(value)) &&
-+      value[0] == L'0') {
++  wchar_t value[MAX_PATH] = {0,};
```

Comment:
Is the comma here required or a typo?

Reply:
I did this intentionally to have same effect with memset with zero.